sbi/user_input/user_input_checks.py

```diff
@@ -527,3 +527,28 @@ def check_estimator_arg(estimator: Union[str, Callable]) -> None:
         "The passed density estimator / classifier must be a string or a function "
         f"returning a nn.Module, but is {type(estimator)}"
     )
+
+
+def test_posterior_net_for_multi_d_x(net: nn.Module, theta: Tensor, x: Tensor) -> None:
+    """Test log prob method of the net.
+
+    This is done to make sure the net can handle multidimensional inputs via an
+    embedding net. If not, it usually fails with a RuntimeError. Here we catch the
+    error, append a debug hint and raise it again.
+    """
+
+    try:
+        # PyTorch needs at least to inputs here.
```

Comment:
to -> two

Reply:
Also, this is not pytorch but nflows, right? And isn't it enough if `theta` and `x` both have a batch dimension, i.e.
```
net.log_prob(theta[:1], x[:1])
```

Reply:
and maybe let's do:
```
_ = net.log_prob(theta[:1], x[:1])
```
i.e. explicitly do not assign

Reply:
No, it's actually a pytorch `nn.functional` error message I get when passing `theta[:1]`, i.e., even with batch dim.